clients/privacy-center/config/config.json

```diff
@@ -30,31 +30,34 @@
   ],
   "includeConsent": true,
   "consent": {
+    "icon_path": "/consent.svg",
+    "title": "Manage your consent",
+    "description": "Manage your consent preferences, including the option to select 'Do Not Sell My Personal Information'.",
     "cookieName": "fides_consent",
     "consentOptions": [
       {
-        "fidesDataUseKey": "provide.service",
-        "name": "Provide our Service",
+        "fidesDataUseKey": "advertising",
+        "name": "Data Sales or Sharing",
+        "description": "We may use some of your personal information for behavioral advertising purposes, which may be interpreted as 'Data Sales' or 'Data Sharing' under regulations such as CCPA, CPRA, VCDPA, etc.",
+        "url": "https://example.com/privacy#data-sales",
```

Comment:
Just trying to continue to make the defaults as realistic as possible, for our own benefit